doc/Programming-guide/Key-platform-features/Data-collection/Flume-log-appender/index.md

```diff
@@ -0,0 +1,424 @@
+---
+layout: page
+title: Flume log appender
+permalink: /:path/
+nav: /:path/Programming-guide/Key-platform-features/Data-collection/Flume-log-appender
+sort_idx: 50
+---
+
+{% assign root_url = page.url | split: '/'%}
+{% capture root_url  %} /{{root_url[1]}}/{{root_url[2]}}/{% endcapture %}
+
+* TOC
+{:toc}
+
+The Flume log appender encapsulates received logs into Flume events and sends these events to external Flume sources via Avro RPC.
+
+# Creating Flume log appender in Admin UI
+
+The easiest way to create a Flume log appender for your application is by using Admin UI.
+
+To create a log appender which will be integrated with Hadoop or Spark, do the following:
```

Comment:
What needs to be changed if the log appender is not considered to be integrated with Hadoop or Spark?

This looks like spare detail and thus should be removed. Otherwise, an additional paragraph should be added covering the opposite case.